src/main/java/com/mixpanel/android/viewcrawler/EditProtocol.java

```diff
@@ -127,7 +137,7 @@ public ViewVisitor readEdit(JSONObject source) throws BadInstructionsException,
                     final JSONArray argPlusType = argsAndTypes.getJSONArray(i);
                     final Object jsonArg = argPlusType.get(0);
                     final String argType = argPlusType.getString(1);
-                    methodArgs[i] = convertArgument(jsonArg, argType);
+                    methodArgs[i] = convertArgument(jsonArg, argType, assetsLoaded);
```

Comment:
Is it a common pattern in JAVA that you pass in an array and modify it in the function, instead of assigning the returned value from the function to the array?